libs/SmartStore/src/com/salesforce/androidsdk/smartstore/app/SalesforceSDKManagerWithSmartStore.java

```diff
@@ -218,6 +223,25 @@ public synchronized void changePasscode(String oldPass, String newPass) {
     }
 
     /**
+     * Returns the database used by smart store in the global context.
+     *
+     * @param dbName The database name. This must be a valid file name without a
+     * 				 filename extension such as ".db". Pass 'null' for default.
+     * @return SmartStore instance.
+     */
+    public SmartStore getGlobalSmartStore(String dbName) {
```

Comment:
Passing in ```null``` or empty string for ```dbName``` here will use the default name. We could expose another method that takes no parameters if we want to, but there are already 4 or 5 of these types of methods here.